src/backend/distributed/deparser/citus_grantutils.c

```diff
@@ -74,7 +74,7 @@ AppendGrantRestrictAndCascade(StringInfo buf, GrantStmt *stmt)
 void
 AppendGrantedByInGrantForRoleSpec(StringInfo buf, RoleSpec *grantor, bool isGrant)
 {
-	if (isGrant && grantor)
+	if (grantor)
```

Comment:
Why do we need to remove isGrant check?

Reply:
To be able to add granted by statement in Revoke statements as well

Reply:
Let's merge this in a separate PR because this looks like continuation of https://github.com/citusdata/citus/pull/7451.